src/redux/reducers/communitiesReducer.js

```diff
@@ -110,6 +111,24 @@ export default function (state = initialState, action) {
           error: action.payload,
         },
       };
+    case UPDATE_SUBSCRIBED_COMMUNITIES:
+      let itemIndex = state.subscribedCommunities.data.findIndex(
```

Comment:
instead of manually updating the list, perhaps we should make use of cache store and keep data well organised...